### DISCUSSION TITLE
[General] Authentication failed

### DISCUSSION THREAD

**Discussion by oguzhaniptes** [ethereum-optimism/developers#628] [General]
### Discussion Topic

Authentication failed
This account has already been linked to another user.

I linked my account to another email, but now I can't access that email, and I'm not sure how to disconnect it. Could you help me with this?

**Comment by hash1go** [comment]
Could you please provide more details about:

1. Which service you're trying to access?
2. What type of account is it?

**Comment by oguzhaniptes** [comment]
1. Im trying access to Superchain Faucet.
2. I had registered my GitHub account with Faucet using an email address, but I've forgotten that email address. Now, I’ve created a new account with a different email, but when I try to log in with GitHub on my new account, I get an error. How can I link my GitHub account to the new email instead?

**Reply by hash1go** [reply]
In other words, are you asking about managing your Github account?
At least, I'm not very familiar with Github accounts, so I recommend utilizing Github Support.

**Reply by joohhnnn** [reply]
While this isn't the typical place for such queries, I'm happy to help: To set up a new email on GitHub, first log into your account and go to "Settings." Select "Emails," then enter your new email address and click "Add." Confirm the new email through the verification link sent to your email. If you want to make it your primary email, select "Primary" next to the email under the "Emails" page. To remove an old email, simply click "Delete" next to it.

**Reply by oguzhaniptes** [reply]
No, I’m not talking about my GitHub account. I registered with another email for Superchain Faucet and I logged in with my current GitHub account. However, I currently cannot access the email I used to register for Superchain, and I cannot associate my GitHub account with the other email. My problem is not with GitHub.
